app/src/main/java/com/imashnake/animite/features/home/MediaSmallRow.kt

```diff
@@ -20,7 +20,7 @@ fun MediaSmallRow(mediaList: List<MediaListQuery.Medium?>, onItemClick: (itemId:
     ) {
         items(mediaList) { media ->
             MediaSmall(
```

Comment:
Are you sure you wanna create rows for null Media? The fallbacks and supporting of nulls doesn't look particularly convincing. I'd probably suggest either an early return on `if (mediaList.isNullOrEmpty()) return` or `mediaList.orEmpty().filterNotNull()` to ensure no Media Item has null fields

Reply:
Wouldn't I have to check every field of `mediaList` to make sure they're not null? I added a `filterNotNull()` anyways.